cnf-certification-test/lifecycle/suite.go

```diff
@@ -273,12 +274,36 @@ func testPodNodeSelectorAndAffinityBestPractices(env *provider.TestEnvironment)
 	testhelper.AddTestResultLog("Non-compliant", badPods, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
 
+func nameInDeploymentSkipList(name, namespace string, list []configuration.SkipScalingTestDeploymentNamesInfo) bool {
+	for _, l := range list {
+		if name == l.Name && namespace == l.Namespace {
+			return true
+		}
+	}
+	return false
+}
+
+func nameInStatefulSetSkipList(name, namespace string, list []configuration.SkipScalingTestStatefulSetNamesInfo) bool {
+	for _, l := range list {
+		if name == l.Name && namespace == l.Namespace {
+			return true
+		}
+	}
+	return false
+}
+
 //nolint:dupl
 func testDeploymentScaling(env *provider.TestEnvironment, timeout time.Duration) {
 	ginkgo.By("Testing deployment scaling")
 	defer env.SetNeedsRefresh()
 	failedDeployments := []string{}
 	for i := range env.Deployments {
+		// Skip deployment if it is allowed by config
+		if nameInDeploymentSkipList(env.Deployments[i].Name, env.Deployments[i].Namespace, env.Config.SkipScalingTestDeploymentNames) {
+			logrus.Debugf("%s is being skipped due to configuration setting", env.Deployments[i].String())
```

Comment:
Looks like an important warning: shouldn't we add this trace in the claimfile with `tnf.ClaimFilePrintf()`?